src/components/daily/parametertooltip/ParameterTooltip.js

```diff
@@ -0,0 +1,116 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React, { PropTypes, PureComponent } from 'react';
```

Comment:
```suggestion
import React from 'react';
import PropTypes from 'prop-types';
```
let's be "propre" directly!

Reply:
If I accept this one, I create an exception compared to other components. 
Lets' do that in a separate PR where we can address all components. 
https://diabeloop.atlassian.net/browse/PT-1146